python/pyspark/ml/classification.py

```diff
@@ -1174,9 +1165,31 @@ def trees(self):
         return [DecisionTreeClassificationModel(m) for m in list(self._call_java("trees"))]
 
 
+class GBTClassifierParams(GBTParams, HasVarianceImpurity):
```

Comment:
this should extend `TreeClassifierParams`

Reply:
@BryanCutler Thanks for your review. 
Seems recently https://github.com/apache/spark/pull/22986 added ```trait HasVarianceImpurity``` and made 
```private[ml] trait GBTClassifierParams extends GBTParams with HasVarianceImpurity```

Reply:
ah, I see. let me take another look..